src/System.ComponentModel.Composition/src/Microsoft/Internal/AttributeServices.cs

```diff
@@ -0,0 +1,43 @@
+// -----------------------------------------------------------------------
```

Comment:
Please replace all these headers with the usual 3 line license comment in all cs and resx files.